<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Área de Conhecimento</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(27, 30, 35);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame_cadastro">
     <property name="minimumSize">
      <size>
       <width>780</width>
       <height>515</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>780</width>
       <height>515</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 59, 72);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_titulo">
      <property name="geometry">
       <rect>
        <x>77</x>
        <y>30</y>
        <width>461</width>
        <height>33</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(220, 225, 236);</string>
      </property>
      <property name="text">
       <string>Componentes Curriculares</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QListView" name="listView">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>75</y>
        <width>640</width>
        <height>300</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>640</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>640</width>
        <height>300</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(86, 138, 242);
border:0px solid;
border-radius:15px;
background-color: rgb(44, 49, 60);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSalvar">
      <property name="geometry">
       <rect>
        <x>155</x>
        <y>395</y>
        <width>120</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(85, 170, 127);
border:0px solid;
border-radius:15px;
}
QPushButton:hover{
border-radius:15px;
border-style:outset;
border-width:1px;
border-color:gray;
background-color: rgb(85, 200, 255);
}
QPushButton:pressed{
border-radius:15px;
border-style:outset;
border-width:1px;
border-color:white;
background-color: rgb(85, 170, 255);
}</string>
      </property>
      <property name="text">
       <string>Salvar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAlterar">
      <property name="geometry">
       <rect>
        <x>315</x>
        <y>395</y>
        <width>120</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius:15px;
border:0px solid;
	background-color: rgb(255, 255, 127);
}
QPushButton:hover{
border-radius:15px;
border-style:outset;
border-width:1px;
border-color:gray;
background-color: rgb(85, 200, 255);
}
QPushButton:pressed{
border-radius:15px;
border-style:outset;
border-width:1px;
border-color:white;
background-color: rgb(85, 170, 255);
}</string>
      </property>
      <property name="text">
       <string>Alterar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnExcluir">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>395</y>
        <width>120</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius:15px;
border:0px solid;
	background-color: rgb(255, 0, 0);
}
QPushButton:hover{
border-radius:15px;
border-style:outset;
border-width:1px;
border-color:gray;
background-color: rgb(85, 200, 255);
}
QPushButton:pressed{
border-radius:15px;
border-style:outset;
border-width:1px;
border-color:white;
background-color: rgb(85, 170, 255);
}</string>
      </property>
      <property name="text">
       <string>Excluir</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>450</y>
        <width>76</width>
        <height>46</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(220, 225, 236);</string>
      </property>
      <property name="text">
       <string>Descrição:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>460</y>
        <width>331</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(44, 49, 60);
color: rgb(86, 138, 242);
border:0px solid;
border-radius:15px;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnListar">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>20</y>
        <width>120</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius:15px;
border:0px solid;
	background-color: rgb(85, 85, 255);
}
QPushButton:hover{
border-radius:15px;
border-style:outset;
border-width:1px;
border-color:gray;
background-color: rgb(85, 200, 255);
}
QPushButton:pressed{
border-radius:15px;
border-style:outset;
border-width:1px;
border-color:white;
background-color: rgb(85, 170, 255);
}</string>
      </property>
      <property name="text">
       <string>Listar</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>460</y>
        <width>211</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(44, 49, 60);
color: rgb(86, 138, 242);
border:0px solid;
border-radius:15px;</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
